.evergreen/config_generator/generate-config.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+# Generates *.yml files under .evergreen/generated_configs.
+#
+# Install dependencies by running (preferably in a virtual environment):
+#     python -m pip install -r .evergreen/config_generator/requirements.txt
+#
+# Invoke this using the command:
+#     PYTHONPATH=".evergreen" python3 .evergreen/config_generator/generate-config.py
+
+
+from glob import iglob
+from importlib import import_module
+from pathlib import Path
+
+
+def all_generators():
+    res = []
+
+    # .evergreen/config_generator/generate_config.py -> .evergreen/config_generator/generators
+    components_dir = Path(__file__).parent / 'generators'
+
+    all_paths = iglob(str(components_dir) + '/**/*.py', recursive=True)
+
+    for path_str in sorted(all_paths):
+        path = Path(path_str)
+        if path == '__init__.py':
+            continue
+
+        generator_path = Path(path_str).relative_to(components_dir)
+        generator_path = str(generator_path)[:-3]  # Drop '.py'.
+        generator_path = generator_path.replace('/', '.')  # 'a/b' -> 'a.b'
+        module_name = f'config_generator.generators.{generator_path}'
+        res.append(import_module(module_name))
```

Comment:
Same comment about `importlib`, but also: `Path.with_suffix('')` can be used to erase the file extension.